### ISSUE TITLE
`FailedCallInfo` printing with `f::Union{typeof(g), typeof(h)}`

### ISSUE THREAD

**Issue by vchuravy** [JuliaDebug/Cthulhu.jl#37]

**Comment by glwagner** [comment]
I got this error

```julia
Cthulhu.TextWidthLimiter(IOContext(IOBuffer(data=UInt8[...], readable=true, writable=true, seekable=true, append=false, size=4, maxsize=Inf, ptr=5, mark=-1)), 4, 215) = call ERROR: LoadError: MethodError: no method matching nameof(::Type{Union{typeof(KS), typeof(KT), typeof(KU)}})
Closest candidates are:
  nameof(::JuliaInterpreter.Frame) at /Users/gregorywagner/.julia/packages/JuliaInterpreter/6LimI/src/utils.jl:18
  nameof(::Module) at reflection.jl:16
  nameof(::DataType) at reflection.jl:197
```

**Comment by vchuravy** [comment]
Fixed in 5d783f1a55762affa65624b1102caf1e1ef37d7d
